pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -302,8 +337,24 @@ func (m *managerImpl) synchronize(diskInfoProvider DiskInfoProvider, podFunc Act
 		}
 	}
 
+	accessibleSwap := uint64(0)
+	if utilfeature.DefaultFeatureGate.Enabled(features.NodeSwap) {
+		if summary != nil && summary.Node.Memory != nil && summary.Node.Swap != nil && summary.Node.Memory.AvailableBytes != nil &&
+			summary.Node.Memory.WorkingSetBytes != nil && summary.Node.Swap.SwapUsageBytes != nil && summary.Node.Swap.SwapAvailableBytes != nil {
+			nodeTotalMemory := *summary.Node.Memory.AvailableBytes + *summary.Node.Memory.WorkingSetBytes
+			nodeTotalSwap := *summary.Node.Swap.SwapAvailableBytes + *summary.Node.Swap.SwapUsageBytes
+			accessibleSwap, err = calcAccessibleSwap(activePods, nodeTotalMemory, nodeTotalSwap)
+			if err != nil {
+				klog.ErrorS(err, "Eviction manager: failed to calculate accessible swap")
```

Comment:
err isn't returned here and there might be a side-effect of implicitly setting accessibleSwap to zero. I suggest we explicitly set accessibleSwap on error here and return (or comment) that the code falls through.

Reply:
Done!